heartbeat.cpp

```diff
@@ -28,4 +32,31 @@ unsigned int heartbeat::getRemainingDuration()
 unsigned int heartbeat::getInterval()
 {
     return HEARTBEAT_INTERVAL;
+}
+
+int heartbeat::setInterval(unsigned int interval) {
+    if (interval > MIN_HEARTBEAT_INTERVAL && interval < MAX_HEARTBEAT_TIMER) {
```

Comment:
What do you think about making this an inclusive range? So >= and <= like with the timers?